Library/ViewModels/PledgeAmountViewModel.swift

```diff
@@ -69,11 +74,8 @@ public final class PledgeAmountViewModel: PledgeAmountViewModelType,
     self.currency = project
       .map { currencySymbol(forCountry: $0.country).trimmed() }
 
-    self.stepperMinValue = minAndMax.signal
-      .map(first)
-
-    self.stepperMaxValue = minAndMax.signal
-      .map(second)
+    self.stepperMinValue = minValue.mapConst(0)
+    self.stepperMaxValue = minValue.mapConst(Double.greatestFiniteMagnitude)
```

Comment:
I've made a decision to use 0...Double.max but we can chose something else as well (I'm open to discussing this).

The reason we're able to do 0...Double.max is because we want to allow the user to actually decrement/increment bellow/above pledge's min/max value...in that case we simply change the input text to red to indicate this out of bounds state.